FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Keep_Alive/DemoTasks/KeepAliveMQTTExample.c

```diff
@@ -367,19 +355,24 @@ static TimerHandle_t xKeepAliveTimer;
 static StaticTimer_t xKeepAliveTimerBuffer;
 
 /**
- * @brief Set to true when PINGREQ is sent then false when PINGRESP is received.
+ * @brief A flag indicating whether a PUBACK from the broker was received.
  */
-static volatile bool xWaitingForPingResp = false;
```

Comment:
Do we document the fact that we're no longer enforcing a time limit on the broker to send a PINGRESP, or that we're not enforcing it to send it at all?

Reply:
I just had an idea to have a separate timer for checking the status of the PINGRESP. Give me a second to implement.